staging/src/k8s.io/component-base/OWNERS

```diff
@@ -13,5 +13,4 @@ emeritus_approvers:
   - sttts
 emeritus_reviewers:
   - dixudx
-  - rosti
```

Comment:
Looks like already emeritus, don't need to remove?

Reply:
good point fixing!